pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/NamespaceOperation.java

```diff
@@ -24,7 +24,6 @@
  */
 public enum NamespaceOperation {
     CREATE_TOPIC,
-    GET_TOPIC,
```

Comment:
unused namespace operation so remove

Reply:
is this a kind of breaking change ? why did we add this value ?

Reply:
I went back a bit, `GET_TOPIC` was originally defined [here](https://github.com/apache/pulsar/pull/6428/files#diff-d7e8390b6795670a5d992eea3966a36e08a572a08d32ea77da370018ea086830R27-R28) but it was not used, and `GET_TOPICS` was used.

Reply:
thanks for your explanation

Reply:
You are welcome :)
Can you take a look at this PR and are there any other questions?

Reply:
Technically, this is a breaking change, and I don't think we should remove the value just yet. Even though it isn't used by the project, extensions of the `AuthorizationProvider` might reference it, and removing the value from the enum would break those implementations. If we think there is no use case for the value, perhaps we could somehow mark it as deprecated and eventually remove it.

@KannarFr - is there a place where the `GET_TOPIC` value was supposed to be used?